packages/shared/src/lib/featureValues.ts

```diff
@@ -20,3 +20,8 @@ export enum OnboardingV4dot5 {
   Control = 'control',
   V4dot5 = 'v4dot5',
 }
+
+export enum PostPageOnboarding {
+  Control = 'control',
+  V2 = 'v2',
```

Comment:
Why is this V2 not V1 🤔

Reply:
I'm not quite sure, because we already had a v1 🤷‍♀️ though to me it makes sense to have different if they run in parallel. That's why have asked here in [thread](https://dailydotdev.slack.com/archives/C06K1LYU716/p1708942468197959?thread_ts=1708931322.030979&cid=C06K1LYU716) and am waiting for confirmation.

Reply:
Oh right, I didn't see it here, but guess it was a experiment winner already?
Maybe we want to be careful and rather use new experiments in that case? 

It might impact already allocated users

Reply:
We already also cleaned up for the v1 experiment, so v1 is already the default in our code. https://github.com/dailydotdev/apps/pull/2681